src/components/targets/CreateTargetForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+import FormInput from '../common/FormInput';
+import { genderSelectStyle } from '../users/genderSelectStyle';
+import FormSelect from '../common/FormSelect';
+
+const CreateTargetForm = ({ intl }) => {
+  const handleSubmit = () => {};
+
+  const { select_topic } = {};
```

Comment:
is this ok?

Reply:
Why do you ask? I use this to maintain a reference to the selected topic

Reply:
- `handleSubmit` is a function without functionality
- `select_topic` is always `undefined`

Reply:
It's missing because I didn't do the communication with the backend. In next PR I will implement `handleSubmit` and `select_topic`